README.md

```diff
@@ -1,61 +1,46 @@
-# Azure Functions Serverless Plugin 
+# Azure Functions Serverless Plugin
 
 This plugin enables Azure Functions support within the Serverless Framework.
 
 [![Coverage Status](https://coveralls.io/repos/github/serverless/serverless-azure-functions/badge.svg)](https://coveralls.io/github/serverless/serverless-azure-functions)
 
-## Getting started
+## Quickstart
 
 ### Pre-requisites
 
 1. Node.js v6.5.0+ *(this is the runtime version supported by Azure Functions)*
```

Comment:
should follow markdown list syntax and use 

```
1. one
1. two
1. three
```

Reply:
The [markdown guide](https://www.markdownguide.org/basic-syntax/#ordered-lists) lists the actual numbered version as the first style, but they both render to the same thing. Also, the VS Code extension auto-generates the next number with a new line.